src/app/QuickStarts/quickstarts/generic-quickstart.tsx

```diff
@@ -21,7 +21,7 @@ import { PficonTemplateIcon } from '@patternfly/react-icons';
 import { conclusion } from '../quickstart-utils';
 
 // Quick start name (currently cannot use [APP], there is a bug with how the title gets rendered in the quick start panel)
-const displayName = 'Getting started with quick starts in Cryostat';
+const displayName = 'Getting started with Quick starts in Cryostat';
```

Comment:
https://github.com/cryostatio/cryostat-web/pull/1260/files#diff-aae9c573fe375d2f91b9cb47cf8795f9779267f70a8e8dfe122f2d3c0024a83aR42

This seems to go against the updated advice in the [src/app/QuickStarts/README.md](https://github.com/cryostatio/cryostat-web/pull/1260/files#diff-aae9c573fe375d2f91b9cb47cf8795f9779267f70a8e8dfe122f2d3c0024a83a)